doc/source/guidelines/doc_practices.rst

```diff
@@ -46,9 +51,9 @@ recommend the use of numpy-style docstrings so that there is consistency
 across PyAnsys libraries. For more information, see :ref:`api_documentation`.
 
 RST Files
----------
+~~~~~~~~~
 To provide general usage information in your documentation, use your favorite editor to
-create RST files that you then place in the repository's ``doc/`` directory. The ``index.rst``
+create RST files that you then place in the repository's ``doc\`` directory. The ``index.rst``
```

Comment:
```suggestion
create RST files that you then place in the repository's ``doc`` directory. The ``index.rst``
```

The `/` works both on Linux, Windows, and MacOS, but `\` will only work on Windows. We can also just remove this character.